src/main/java/io/github/xmppinteroptesting/bambootask/ExecuteInteropTestsTask.java

```diff
@@ -81,10 +83,16 @@ public TaskResult execute(@NotNull final TaskContext taskContext) throws TaskExc
                 command.add("-Dsinttest.adminAccountPassword=" + accountPassword);
             }
             if (!StringUtils.isBlank(disabledTests)) {
-                command.add("-Dsinttest.disabledTests=" + accountUsername);
+                command.add("-Dsinttest.disabledTests=" + disabledTests);
```

Comment:
Eek. Thanks for catching this. Let's fire the one that was responsible for this. From a cannon.